index.html

```diff
@@ -0,0 +1,288 @@
+                        </div>
+                    </div>
+                    <div class="speakerInfo">
+                        <div class="SepeakerPicture">
+                            <img src="./images/speaker_background.jpg" class="speakerBackground" alt="speakerBackground">
+                        <img src="./images/pexels-andrea-piacquadio-3769021.jpg"  class="speakerImg" alt="speakerImg">
+                        </div>
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Andrea Piaquadio</h2>
+                            <p class="SpeakerPosition">Project Manager</p>
+                            <p class="SpeakerDescription">Say hello to Andrea, a skilled project manager in the realm of e-commerce. With a knack for organization and a strategic mindset, Andrea ensures successful implementation and delivery of projects in the digital marketplace.</p>
+                        </div>
+                    </div>
+                    <div class="speakerInfo">
+                        <div class="SepeakerPicture">
+                            <img src="./images/speaker_background.jpg" class="speakerBackground" alt="speakerBackground">
+                        <img src="./images/pexels-emmy-e-2381069.jpg"  class="speakerImg" alt="speakerImg">
+                        </div>
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Alex Wolf</h2>
+                            <p class="SpeakerPosition">Human Recourses</p>
+                            <p class="SpeakerDescription">Meet Alex, a dedicated human resources professional specializing in the fast-paced world of e-commerce. With a focus on talent acquisition and employee development, Alex cultivates a thriving and motivated workforce within the industry.</p>
+                        </div>
+                    </div>
+                    <div class="speakerInfo">
+                        <div class="SepeakerPicture">
+                            <img src="./images/speaker_background.jpg" class="speakerBackground" alt="speakerBackground">
+                        <img src="./images/pexels-christina-morillo-1181690.jpg"  class="speakerImg" alt="speakerImg">
+                        </div>
+                        <div class="SpeakerCurr">
+                            <h2 class="SpeakerTitle">Christina Morillo</h2>
+                            <p class="SpeakerPosition">Finances</p>
+                            <p class="SpeakerDescription">Introducing Christina, a dynamic professional specializing in business deployment within the realm of e-commerce. With a strategic mindset and meticulous planning, Christina drives successful implementation of business strategies in the digital marketplace.</p>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] NIce so far: as per project requirement, you are not to render the future speaker on the index pages you are expected to render it f dynamically from your javascript using the array method. i kindly suggest your render by dynamically from javascript

Reply:
ok. I restarted again in speakers section. Now is working with DOM